lively.merger/README.md

```diff
@@ -1,3 +1,24 @@
 # lively.merger
 
-No description for package lively.merger yet.
+## Entry Point
+
+The entry point is the merger class. In this the different functionality for merging morphs is described.
```

Comment:
It's not really a class, is it? Also, I think it's cool to have links to the corresponding files

```suggestion
The main entry point is the [merger](./merger.js). Here, the different functionality for merging morphs is described.

```